.github/workflows/lint.yaml

```diff
@@ -24,7 +24,7 @@ jobs:
       image: ghcr.io/gravitational/teleport-buildbox:teleport15
```

Comment:
While you're here, shouldn't this use the `teleport16` tag now?

Reply:
Sure:

* https://github.com/gravitational/teleport.e/pull/3405
* https://github.com/gravitational/teleport/pull/37994